lib/multiplexer/tls.go

```diff
@@ -141,7 +141,7 @@ func (l *TLSListener) detectAndForward(conn *tls.Conn) {
 	start := l.cfg.Clock.Now()
 	if err := conn.Handshake(); err != nil {
 		if trace.Unwrap(err) != io.EOF {
-			l.log.WithError(err).Warning("Handshake failed.")
+			l.log.WithError(err).Debug("Handshake failed.")
```

Comment:
Does this not risk obscuring issues related to the validity of certificates etc if the user is not running with debug log levels enabled, or are those emitted elsewhere?